installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -0,0 +1,134 @@
+[id="manually-creating-iam"]
+= Manually creating IAM
+include::modules/common-attributes.adoc[]
+:context: manually-creating-iam
+
+toc::[]
+
+[id="manually-create-iam_{context}"]
+== Manually create IAM
+
+The Cloud Credential Operator can be put into manual mode prior to installation
+in environments where the cloud identity and access management (IAM) APIs are
+not reachable, or the administrator prefers not to store an admin level
+credential secret in the cluster `kube-system` namespace.
+
+.Procedure
+
+. Run the {product-title} installer to generate manifests:
++
+----
+$ openshift-install create manifests --dir=mycluster
+----
+
+. Insert a ConfigMap into the manifests directory so that the Cloud Credential
+Operator will be be placed in manual mode:
++
+----
+$ cat <<EOF > mycluster/manifests/cco-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-credential-operator-config
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    release.openshift.io/create-only: "true"
+data:
+  disabled: "true"
+EOF
+----
```

Comment:
By the time 4.6 launches there'll be a better and new mechanism for doing this using an actual config resource that no longer calls it disabled, rather manual mode. 

Prior to 4.6 however the above configmap will be the only option, so if we backport this for 4.5 (where manual mode does work) we would need to change this portion.

cc @joelddiaz

Reply:
What @dgoodwin is refering to here is the enhancement proposal ( https://github.com/openshift/enhancements/blob/master/enhancements/installer/aws-permissions-check-bypass.md ) to allow bypassing permissions checks before install. A side-effect of enabling this functionality is that we will also (using the same proposed CCO config resource) be able to force CCO into "manual" mode w/o the need for this configmap.

Reply:
@dgoodwin @joelddiaz Once this PR merges with the 4.5 content, we can do a follow-up with 4.6 content if that works for you.